### PR TITLE
fix(PopScope): User able to exit patch screen when the installer is still running

### PR DESCRIPTION
**Pull request by BenjaminHalko**
fixes: #1645

### CODE REVIEW

**Comment by validcube** [comment]
Please rebase this on the latest commit.

File: lib/ui/views/installer/installer_view.dart [hunk 1]
```diff
@@ -17,6 +17,8 @@ class InstallerView extends StatelessWidget {
       onViewModelReady: (model) => model.initialize(context),
       viewModelBuilder: () => InstallerViewModel(),
       builder: (context, model, child) => PopScope(
+        onPopInvoked: (bool didPop) => model.onWillPop(context),
+        canPop: false,
```
Comment (validcube) [inline lib/ui/views/installer/installer_view.dart:21]: The code seems skeptical to me.

Can you exit out of the screen when you finished patching?

Comment (BenjaminHalko) [reply lib/ui/views/installer/installer_view.dart:21]: Yep, it does work. At least on my device.

Comment (validcube) [reply lib/ui/views/installer/installer_view.dart:21]: > Yep, it does work. At least on my device.

I'll revisit this later.:tm:

**Pull request merged**